### DISCUSSION TITLE
How to change which GPU to process on?

### DISCUSSION THREAD

**Discussion by IcyAlmond** [catsout/wallpaper-engine-kde-plugin#176] [Q&A] [answered]
When I ran a wallpaper with the parallax effect, it pegged my rx 550 to 100%. I'm not sure if it's because the code isn't efficient yet or if the GPU is too weak. But I wanted to try running it my other GPU and I can't find out how to.

**Comment by catsout** [comment, marked as answer]
it's possible to implement this in vulkan, but it's low priority.

**Reply by IOplus** [reply]
I'm waiting for this. Running on a hybrid graphics laptop here, and the integrated GPU can't handle much.
